src/elements/asset.rs

```diff
@@ -178,11 +178,17 @@ pub fn index_confirmed_tx_assets(
     network: Network,
     parent_network: BNetwork,
     rows: &mut Vec<DBRow>,
+    op: &Operation,
 ) {
     let (history, issuances) = index_tx_assets(tx, network, parent_network);
 
     rows.extend(history.into_iter().map(|(asset_id, info)| {
-        asset_history_row(&asset_id, confirmed_height, tx_position, info).into_row()
+        let history_row = asset_history_row(&asset_id, confirmed_height, tx_position, info);
```

Comment:
I think maybe we should also be sending over the issuances that are being added below this diff...?

The comment seems to indicate that the issuances DBRows are redundant (they also exist as history entries)...

Reply:
yes, probably.

the `b"i"` issuance rows are used in the `lookup_asset()` method, so otherwise I think we'd leave behind phantom assets in the event that an initial issuance transaction got reorged out but not re-confirmed in a subsequent block.

Reply:
on the other hand, can Liquid even have reorgs? 😅